CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2719,12 +2719,16 @@ end
 -- Permissions --
 -----------------
 do
-	local permissions;
+	local permissionsLoading, permissions = false;
```

Comment:
if I'm not mistaken, I don't think permissionsLoading is initialized here, although it starts as nil, which is very similar to false.